doc/nrf/templates/application_README.rst

```diff
@@ -0,0 +1,260 @@
+.. _application:
+
+Template: Application
+#####################
+
+.. contents::
+   :local:
+   :depth: 2
+
+.. note::
+   * Give the application a concise name that corresponds to the folder name.
+     If the application targets a specific device, add it in the title before the application name (for example, "nRF9160:").
+     Do not include the word "application" in the title.
+   * Put the documentation inside the :file:`applications` folder and use the file name :file:`README.rst`.
+   * Use the provided stock phrases and includes when possible.
+   * Sections with * are optional and can be left out.
+     All other sections are required.
+     Do not add new sections (unless in the sections that allow for further subsections) without discussion with the tech writer team.
+
+The Application application demonstrates the bridge functionality for the Thingy:91 hardware.
```

Comment:
"Application" implies the name of the application here.

Reply:
May be we could mention as : "The Connectivity bridge application demonstrates the bridge functionality for the Thingy:91 hardware. " just to avoid the 2 "application" here?

https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-9360/nrf/applications/connectivity_bridge/README.html

Reply:
This is done not to specify any applications name and keep it general. Also, to be consistent with the sample template as well: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/templates/sample_README.html

Reply:
Changed to "XYZ" application as a possible solution.